Worked example: multiple tags (#26)
open-contracting/sphinxcontrib-opencontracting: docs/index.rst

Where:

> - The ``workedexample`` directive is used at the top of a section to mark the content as a worked example. Use the ``tags`` attribute to indicate the content this worked example is referring to. Multiple values are allowed by using comma-separated values. Its argument is used as the text of the hyperlink rendered by the ``workedexamplelist`` directive.

Reviewer: ```suggestion
- The ``workedexample`` directive is used at the top of a section to mark the content as a worked example. Its argument is used as the text of the hyperlink rendered by the ``workedexamplelist`` directive. Tag the worked example with one or more comma-separated tags, using the ``tags`` option.
```
